http-client/src/test/java/io/airlift/http/client/TestHttpUriBuilder.java

```diff
@@ -118,6 +119,15 @@ public void testAppendPathElidesSlashes()
     }
 
     @Test
+    public void testNullPath()
+            throws URISyntaxException
+    {
+        URI uri = new URI("test-server:8080");
```

Comment:
Making the path `""` actually causes another failure:
```
Caused by: java.net.URISyntaxException: Expected authority at index 14: test-server://
```
So this is much more involved :(